lab_1/main.py

```diff
@@ -87,7 +160,32 @@ def get_adjacent_words(tokens: list, word: str, left_n: int, right_n: int) -> li
     right_n = 3
     --> [['man', 'is'], ['dog, 'cat']]
     """
-    pass
+    conco_all = []
+    index_word = 0
+    if checker(tokens, list) and checker(word, str) \
+            and checker(left_n, int) and checker(right_n, int) \
+            and left_n >= 0 and right_n >= 0:
+
+        for i in range(tokens.count(word)):
+            if tokens.index(word) == 0:
```

Comment:
No need to check it explicitly.

Reply:
I get a mistake without it

Reply:
I mean, you can do just:
`if not tokens.index(word)` which results in `True`, just like your line.